docs/Rendering-Ads.md

```diff
@@ -0,0 +1,34 @@
+# Rendering Ad
+
+We are currently rendering two types of Ads on our front pages.
+
+Leaderboard Ad
+![Leaderboard Ad
+](https://user-images.githubusercontent.com/30599794/90142128-eb7c7080-dd73-11ea-8c68-f47b27f56620.png)
+
+MPU Ad
+![MPU Ad](https://user-images.githubusercontent.com/30599794/90142048-cee03880-dd73-11ea-92f7-1f1c9098d228.png)
+
+## Services with Ads enabled
+
+Afrique, Arabic, Hindi, Mundo, Portuguese, Russian, Zhongwen
+
+## Rendering Ads on local environment
+
+Given you are using Chrome browser.
+
+1. Download [ModHeader Chrome extention](https://www.google.com/search?q=modheader+chrome+pass+boolean&rlz=1C5CHFA_enGB762GB762&oq=modheader&aqs=chrome.0.69i59l2j69i57j69i60j69i61j69i60j69i65l2.1039j0j7&sourceid=chrome&ie=UTF-8)
+2. With the ModHeader extenion, add a header of `BBC-Adverts` with the value `true`.
+
+   i.e: ![image](https://user-images.githubusercontent.com/30599794/90151074-40bd7f80-dd7e-11ea-985d-902ed04641ac.png)
+
+3. Run local environment (`cd simorgh && npm run dev`)
```

Comment:
Do we need to mention anything about running a production build to apply the CSP headers?

Reply:
Yeah, I think it is worth adding.

Reply:
I've added a step here: https://github.com/bbc/simorgh/pull/7489/commits/cc944133ebc5214f9991f8a15271f21ddf14b557

hope it is correct 👀